src/app-layer-htp-libhtp.h

```diff
@@ -36,7 +36,11 @@
  ***************************************************************************/
 
 /**
- * Anoop Saldanha <anoopsaldanha@gmail.com>
+ *\file 
+ * 
+ * APIs from libhtp 0.5.x.
+ *
+ * \author Saldanha <anoopsaldanha@gmail.com>
```

Comment:
incomplete name, please review all

Reply:
I was going for URL Normalization function for HTTP traffic library but Juliana pointed out that the naming should be more generic so I opted for the APIs from libhtp 0.5.x. in line will her suggestion.
I'm not certain how to proceed from there. Kindly advice.

Reply:
I was going for URL Normalization function for HTTP traffic library but Juliana pointed out that the naming should be more generic so I opted for the APIs from libhtp 0.5.x. in line will her suggestion.
I'm not certain how to proceed from there. Kindly advice.